Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -98,7 +98,13 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
        .. code-block:: shell-session
 
           minikube start --network-plugin=cni
-          minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
+
+.. note::
+
+   From minikube v1.12.1+, cilium networking plugin can be enabled directly with
```

Comment:
Let's bump the minikube version in line 95 to v1.12.1 and remove this note altogether.

Reply:
Yeah, thought about it. Sounds good to me!